ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -255,23 +257,85 @@ export abstract class BaseNode<
     });
   }
 
+  protected getRetryAdornment(): AdornmentNode | undefined {
+    if (this.nodeData.adornments && this.nodeData.adornments.length > 0) {
+      const retryAdornment = this.nodeData.adornments.find(
+        (adornment) => adornment.base && adornment.base.name === "RetryNode"
+      );
+
+      if (retryAdornment && retryAdornment.attributes) {
+        const maxAttemptsAttr = retryAdornment.attributes.find(
+          (attr) => attr.name === "max_attempts"
```

Comment:
[oos] In fact, I could see us creating adornment generators that could use the specific schema for each node class for specific attribute validation/generation like this

Reply:
But for now, I think we just try to generate the attribute no matter which one is found